changelog/3.7.0/3.7.0.md

```diff
@@ -0,0 +1,394 @@
+ [KAG-3653](https://konghq.atlassian.net/browse/KAG-3653)
+
+- Bumped PCRE from the legacy libpcre 8.45 to libpcre2 10.43
+ [#12366](https://github.com/Kong/kong/issues/12366)
+ [KAG-3571](https://konghq.atlassian.net/browse/KAG-3571) [KAG-3521](https://konghq.atlassian.net/browse/KAG-3521) [KAG-2025](https://konghq.atlassian.net/browse/KAG-2025)
+
+- Bumped penlight to 1.14.0
+ [#12862](https://github.com/Kong/kong/issues/12862)
+
+#### Default
+
+- Add package `tzdata` to DEB Docker image for convenient timezone setting.
+ [#12609](https://github.com/Kong/kong/issues/12609)
+ [FTI-5698](https://konghq.atlassian.net/browse/FTI-5698)
+
+- Bump lua-resty-http to 0.17.2.
+ [#12908](https://github.com/Kong/kong/issues/12908)
+
+
+- Bumped LuaRocks from 3.9.2 to 3.11.0
+ [#12662](https://github.com/Kong/kong/issues/12662)
+ [KAG-3883](https://konghq.atlassian.net/browse/KAG-3883)
+
+- Bump `ngx_wasm_module` to `91d447ffd0e9bb08f11cc69d1aa9128ec36b4526`
+ [#12011](https://github.com/Kong/kong/issues/12011)
+
+
+- Bump `V8` version to `12.0.267.17`
+ [#12704](https://github.com/Kong/kong/issues/12704)
+
+
+- Bump `Wasmtime` version to `19.0.0`
+ [#12011](https://github.com/Kong/kong/issues/12011)
+
+
+- Improve the robustness of lua-cjson when handling unexpected input.
+ [#12904](https://github.com/Kong/kong/issues/12904)
+ [KAG-4275](https://konghq.atlassian.net/browse/KAG-4275)
+
+### Features
+#### Configuration
+
+- now TLSv1.1 and lower is by default disabled in OpenSSL 3.x
+ [#12420](https://github.com/Kong/kong/issues/12420)
+ [KAG-3259](https://konghq.atlassian.net/browse/KAG-3259)
+
+- Introduce `nginx_wasm_main_shm_kv` configuration entry, which enables
+Wasm filters to use the Proxy-Wasm operations `get_shared_data` and
+`set_shared_data` without namespaced keys.
+ [#12663](https://github.com/Kong/kong/issues/12663)
+
+
+- **Schema**: Added a deprecation field attribute to identify deprecated fields
+ [#12686](https://github.com/Kong/kong/issues/12686)
+ [KAG-3915](https://konghq.atlassian.net/browse/KAG-3915)
+
+- Add `wasm_filters` configuration value for enabling individual filters
+ [#12843](https://github.com/Kong/kong/issues/12843)
+ [KAG-4211](https://konghq.atlassian.net/browse/KAG-4211)
+#### Core
+
+- Add `events:ai:response_tokens`, `events:ai:prompt_tokens` and `events:ai:requests` to the anonymous report to start counting AI usage
+ [#12924](https://github.com/Kong/kong/issues/12924)
+
+
+- When CP runs with `expressions` flavor:
+- if mixed config is detected and a lower DP is attached to the CP, no config will be sent at all
+- if the expression is invalid on CP, no config will be sent at all
+- if the expression is invalid on lower DP, it will be sent to the DP and DP validation will catch this and communicate back to the CP (this could result in partial config application)
+ [#12967](https://github.com/Kong/kong/issues/12967)
+ [KAG-3806](https://konghq.atlassian.net/browse/KAG-3806)
```

Comment:
```suggestion
- Improved config handling when the CP runs with the router set to the `expressions` flavor:
  - If mixed config is detected and a lower DP is attached to the CP, no config will be sent at all
  - If the expression is invalid on the CP, no config will be sent at all
  - If the expression is invalid on a lower DP, it will be sent to the DP and DP validation will catch this and communicate back to the CP (this could result in partial config application)
 [#12967](https://github.com/Kong/kong/issues/12967)
 [KAG-3806](https://konghq.atlassian.net/browse/KAG-3806)
```